src/System.IO.FileSystem/tests/System.IO.FileSystem.Tests.builds

```diff
@@ -2,25 +2,23 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
+    <Project Include="Performance\System.IO.FileSystem.Performance.Tests.csproj" />
```

Comment:
nit: This .builds file has interesting ordering because it contains multiple build projects. If I were to sort this one I would still group the individual test projects. I realize you generated this so I know how it turned out like this I just wanted to point it out to you.